README.md

```diff
@@ -122,7 +122,7 @@ Happy programming :smiley: :v:
 
 Icons from:
 
-[![Icons8][icons8-logo]][Icons8] Icons8
```

Comment:
@Israel-Laguan the link should go to this page?

![image](https://user-images.githubusercontent.com/66505715/169299993-436122c4-dbf2-4f1f-9d09-09bb551a1d1c.png)

Reply:
@Dajava02 maybe you could use the Icons8 variable on line 152

https://github.com/Israel-Laguan/parrot-echo/blob/eebb7e843b7f597c7a51e9ab1f7ad08697a9e2a8/README.md?plain=1#L151

You can follow this doc about how to use it https://thehotcode.com/markdown-variables/

Reply:
I think that's fine then. 😁

Reply:
I think that's fine then. 😁